AngularVReact.md

```diff
@@ -20,42 +20,32 @@ Angular is full MVC while React is simply V.
 
 - Watchers: Watchers (similar to event listeners) are attached to each component and each time a component is changed, watchers check if we should modify something else; and if needed, make appropriate modifications. This part in Angular 2 is way fast than its previous version so each time a component is changed, we don’t have to run any verifications on objects (depending on immutable elements).
 
-- Typescript: Angular2 requires Typescript. Compared to vanillaJS, Typescript offers better code organization, typing, and annotations so Typescript is better from a 'strictness' perspective but in regards to 'learning' you will often find yourself learning Typescript and Angular at the same time.
-
```

Comment:
I think we should keep this sentence, but instead of saying "Angular2 requires Typescript`, we can change it to "Angular2 comes default with Typescript". I think it is beneficial to include some of the benefits of Typescript over vanilla jS.